.gitmodules

```diff
@@ -1,9 +1,16 @@
 [submodule "packages/contracts/lib/ds-test"]
 	path = packages/contracts/lib/ds-test
 	url = https://github.com/dapphub/ds-test
+    ignore = dirty
```

Comment:
tabs vs spaces issue here. You set `ignore = dirty` for debugging so that you could make changes to the files without git picking up on the changes? If so, I think that we don't want that set for the long term